app/editor/src/features/admin/actions/ActionList.tsx

```diff
@@ -59,6 +59,7 @@ export const ActionList: React.FC = () => {
           columns={columns}
           showSort={true}
           onRowClick={(row) => navigate(`${row.original.id}`)}
+          pagingEnabled={false}
```

Comment:
We now have a property to turn off paging.  When this is false it will place all items on the page.